includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -1330,4 +1330,23 @@ public function create_setup_intent_ajax() {
 			);
 		}
 	}
+
+	/**
+	 * Add a url to the admin order page that links directly to the transactions detail view.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @param WC_Order $order The context passed into this function when the user view the order details page in WordPress admin.
+	 */
+	public function get_transaction_url( $order ) {
+		$charge_id = $order->get_meta( '_charge_id' );
```

Comment:
@dwainm does it worth checking that `charge_id` is set and fallback to null if it's not? This scenario shouldn't happen often but would avoid broken link in case order is somehow corrupted.

Reply:
Good catch added in 1fd2228.